doc/source/conf.py

```diff
@@ -0,0 +1,62 @@
+# Configuration file for the Sphinx documentation builder.
```

Comment:
We use `black` now for the message-ix stack. I've modified the corresponding files:
- Extend the `# ---` separators to column 88.
- Rewrap the comments to use the full width.
- Blacken, e.g. use `"` instead of `'`.

See e.g. the ixmp conf.py: https://github.com/iiasa/ixmp/blob/master/doc/conf.py

Reply:
Done, and learned something...